File: Leet Code/Python/problem_28.py
class Solution(object):
    def strStr(self, haystack, needle):
        """
        :type haystack: str
        :type needle: str
        :rtype: int
        """
        return haystack.find(needle)
    

sl = Solution()
haystack = input("Enter haystack: ")
needle = input("Enter needle: ")
index = sl.strStr(haystack, needle)
print(index)

File: Leet Code/Java/Problem28.java
public class Problem28 {
    public static void main(String[] args) {
        int result = strStr("sadbudsad", "sad");
        System.out.println(result);
    }

    static int strStr(String haystack, String needle) {
        int index = haystack.indexOf(needle);
        return index;
    }
}
